lib/helpers.js

```diff
@@ -12,40 +11,10 @@ function deferMethod() {
   return setImmediate
 }
 
-function constructObject(initialObject) {
```

Comment:
Nice! +1 on removing this custom implementation and using normal javascript objects